virttest/env_process.py

```diff
@@ -1297,6 +1303,9 @@ def preprocess(test, params, env):
             if libvirtd_inst is None:
                 libvirtd_inst = utils_libvirtd.Libvirtd()
             libvirtd_inst.restart()
+            if virtqemud_inst is None:
```

Comment:
you may add one factory method in utils_split_daemons.py
    def create_daemon_instance():
        if libvirtd_decorator.get_libvirt_version_compare(5, 6, 0) and
               libvirtd_decorator.get_libvirtd_split_enable_bit():
            return utils_split_daemons.VirtQemud()
        else:
            return  utils_libvirtd.Libvirtd()

libvirtd_inst = create_daemon_instance()
libvirtd_inst.restart()